### PR TITLE
Move fields from introspect serializer to user profile

### PR DESCRIPTION
**Pull request by dickdavis**
## Summary

This PR moves user-specific fields from the introspect serializer to the user profile.

## Related issue(s)

https://app.zenhub.com/workspaces/identity-5f5bab705a94c9001ba33734/issues/gh/department-of-veterans-affairs/va.gov-team/79058

## Testing done

* Start vets-website and vets-api
* Login as a user.
* Open the response from the user endpoint in the network tab.
* Verify the following fields are present in the `profile` property:
  * `icn`
  * `birls_id`
  * `edipi`
  * `sec_id`
  * `logingov_uuid`
  * `idme_uuid`
  * `id_theft_flag`
* Verify the following fields are present in the `va_profile` property:
  * `cerner_id`
  * `cerner_facility_ids`
  * `active_mhv_ids`
* Verify the following fields are present in the `vet360_contact_information` property:
  * `vet360_id`

### Example response

```json
{
  "data": {
    "id": "",
    "type": "users_scaffolds",
    "attributes": {
      "services": [
        "facilities",
        "hca",
        "edu-benefits",
        "form-save-in-progress",
        "form-prefill",
        "evss-claims",
        "lighthouse",
        "form526",
        "user-profile",
        "appeals-status",
        "identity-proofed",
        "vet360"
      ],
      "account": {
        "account_uuid": "bf115f6f-e305-4bfc-967b-8a584e2e7db2"
      },
      "profile": {
        "email": "vets.gov.user+1@gmail.com",
        "first_name": "Greg",
        "middle_name": "A",
        "last_name": "Anderson",
        "birth_date": "1933-04-05",
        "gender": "M",
        "zip": "75014",
        "last_signed_in": "2024-03-27T18:20:10.932Z",
        "loa": {
          "current": 3,
          "highest": 3
        },
        "multifactor": true,
        "verified": true,
        "sign_in": {
          "service_name": "idme",
          "auth_broker": "sis",
          "client_id": "vaweb"
        },
        "authn_context": "http://idmanagement.gov/ns/assurance/loa/3",
        "inherited_proof_verified": false,
        "claims": null,
        "icn": "1008596379V859838",
        "birls_id": "796121200",
        "edipi": [
          "1005079124"
        ],
        "sec_id": "0000028114",
        "logingov_uuid": null,
        "idme_uuid": "85c50aa76934460c8736f687a6a30546",
        "id_theft_flag": false
      },
      "va_profile": {
        "status": "OK",
        "birth_date": "19330405",
        "family_name": "Anderson",
        "gender": "M",
        "given_names": [
          "Greg",
          "A"
        ],
        "is_cerner_patient": false,
        "cerner_id": null,
        "cerner_facility_ids": [],
        "facilities": [
          {
            "facility_id": "757",
            "is_cerner": false
          }
        ],
        "va_patient": true,
        "mhv_account_state": "NONE",
        "active_mhv_ids": []
      },
      "veteran_status": null,
      "in_progress_forms": [],
      "prefills_available": [
        "21-686C",
        "40-10007",
        "0873",
        "22-1990",
        "22-1990N",
        "22-1990E",
        "22-1990EMEB",
        "22-1995",
        "22-5490",
        "22-5490E",
        "22-5495",
        "22-0993",
        "22-0994",
        "FEEDBACK-TOOL",
        "22-10203",
        "22-1990S",
        "22-1990EZ",
        "21-526EZ",
        "1010ez",
        "10-10EZR",
        "21P-530",
        "21P-527EZ",
        "21P-530V2",
        "686C-674",
        "20-0995",
        "20-0996",
        "10182",
        "MDOT",
        "5655",
        "28-8832",
        "28-1900",
        "26-1880",
        "26-4555",
        "21-0966"
      ],
      "vet360_contact_information": null,
      "session": {
        "auth_broker": "sis",
        "ssoe": false,
        "transactionid": null
      }
    }
  },
  "meta": {
    "errors": [
      {
        "external_service": "VAProfile",
        "start_time": "2024-03-27T18:20:24Z",
        "end_time": null,
        "description": "Breakers::OutageException, Outage detected on VAProfile/ContactInformation beginning at 1711563611, Breakers::OutageException",
        "status": 503
      },
      {
        "external_service": "VAProfile",
        "start_time": "2024-03-27T18:20:24Z",
        "end_time": null,
        "description": "Breakers::OutageException, Outage detected on VAProfile/MilitaryPersonnel beginning at 1711563611, Breakers::OutageException",
        "status": 503
      }
    ]
  }
}
```

## Screenshots

Not relevant.

## What areas of the site does it impact?

The user endpoint.

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by bramleyjl — DISMISSED** [review]
Confirmed `profile`, `va_profile`, & `vet360_contact_information` have been updated with user ids.

File: app/services/users/profile.rb [hunk 1]
```diff
@@ -79,9 +80,17 @@ def profile
         sign_in: user.identity.sign_in,
         authn_context: user.authn_context,
         inherited_proof_verified: user.inherited_proof_verified,
-        claims:
+        claims:,
+        icn: user.icn,
+        birls_id: user.birls_id,
+        edipi: user.mpi_edipis,
```
Comment (bosawt) [inline app/services/users/profile.rb:86]: can we just call `user.edipi` here?

**Review by bosawt — CHANGES_REQUESTED** [review]
Only had one small comment for a quick change but otherwise looks good

**Review by bosawt — APPROVED** [review]
looks good, I authenticated and inspected `/user` and confirmed the new fields

**Pull request merged**